<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>registro_user</class>
 <widget class="QDialog" name="registro_user">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>330</width>
    <height>470</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>330</width>
    <height>470</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>330</width>
    <height>470</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #f5f6fa;</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="widgetPrincipal" native="true">
     <property name="layoutDirection">
      <enum>Qt::LayoutDirection::LeftToRight</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget{
	background-color: #F7F9FC;
	color: #2d2d2d;
}
QLineEdit {
    border: 2px solid #2980b9;
    border-radius: 12px;
    padding: 2px 5px;
    background-color: white;
	color: #2d2d2d;
}
QPushButton {
	background-color: #2980b9;
    color: #FFFFFF;
    border: none;
    padding: 8px 10px;
    border-radius: 14px;
}
QPushButton:hover {
	background-color: #0D47A1
}</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item alignment="Qt::AlignmentFlag::AlignHCenter">
       <widget class="QLabel" name="lblTitulo_login">
        <property name="minimumSize">
         <size>
          <width>241</width>
          <height>21</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>250</width>
          <height>21</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Segoe UI</family>
          <pointsize>12</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LayoutDirection::LeftToRight</enum>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>Registro de usuario</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignmentFlag::AlignHCenter">
       <widget class="QLineEdit" name="lneNombreCompleto_reg_user">
        <property name="minimumSize">
         <size>
          <width>225</width>
          <height>35</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>225</width>
          <height>35</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Segoe UI</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="mouseTracking">
         <bool>true</bool>
        </property>
        <property name="focusPolicy">
         <enum>Qt::FocusPolicy::ClickFocus</enum>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
        <property name="placeholderText">
         <string>NOMBRE COMPLETO</string>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignmentFlag::AlignHCenter">
       <widget class="QLineEdit" name="lneUsername_reg_user">
        <property name="minimumSize">
         <size>
          <width>225</width>
          <height>35</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>225</width>
          <height>35</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Segoe UI</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="focusPolicy">
         <enum>Qt::FocusPolicy::ClickFocus</enum>
        </property>
        <property name="maxLength">
         <number>8</number>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
        <property name="placeholderText">
         <string>USUARIO</string>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignmentFlag::AlignHCenter">
       <widget class="QLineEdit" name="lnePass_reg_user">
        <property name="minimumSize">
         <size>
          <width>225</width>
          <height>35</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>225</width>
          <height>35</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Segoe UI</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="focusPolicy">
         <enum>Qt::FocusPolicy::ClickFocus</enum>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
        <property name="placeholderText">
         <string>CONTRASEÑA</string>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignmentFlag::AlignHCenter">
       <widget class="QLineEdit" name="lneRepPass_reg_user">
        <property name="minimumSize">
         <size>
          <width>225</width>
          <height>35</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>225</width>
          <height>35</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Segoe UI</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="focusPolicy">
         <enum>Qt::FocusPolicy::ClickFocus</enum>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
        <property name="placeholderText">
         <string>REPITA CONTRASEÑA</string>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignmentFlag::AlignHCenter">
       <widget class="QFrame" name="frameRol_login">
        <property name="minimumSize">
         <size>
          <width>225</width>
          <height>35</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>225</width>
          <height>35</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Segoe UI</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QFrame{
	background-color: white;
	border: 1.5px solid #1e88e5;
	border-radius: 12px;
}
QComboBox{
    border-radius: 10px;
    background-color: white;
    color: #333;
}
</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::Shape::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Shadow::Raised</enum>
        </property>
        <widget class="QComboBox" name="cbxRol_reg_user">
         <property name="geometry">
          <rect>
           <x>11</x>
           <y>5</y>
           <width>201</width>
           <height>26</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>201</width>
           <height>26</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>201</width>
           <height>26</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Segoe UI</family>
           <pointsize>13</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QComboBox {
			combobox-popup: 0;
            border-radius: 10px;
            padding: 2px 8px;
            background-color: white;
            color: #2d2d2d;
   
        }
        QComboBox:hover {
            border-color: #0D47A1;
        }
        
        QComboBox QAbstractItemView {
            background-color: white;
            border: 1.5px solid #2980b9;
            border-radius: 8px;
            color: #2d2d2d;
            outline: none;
            padding: 4px 0px;
        }
        QComboBox QAbstractItemView::item {
            min-height: 20px;
            padding: 3px 5px;
        }
        QComboBox QAbstractItemView::item:selected {
            background-color: #2980b9;
            color: white;
        }
        QComboBox QAbstractItemView QScrollBar:vertical {
            border: none;
            background: #f0f7ff;
            width: 7px;
            border-radius: 3px;
            margin: 4px 2px;
        }
        QComboBox QAbstractItemView QScrollBar::handle:vertical {
            background: #2980b9;
            border-radius: 3px;
            min-height: 20px;
        }
        QComboBox QAbstractItemView QScrollBar::add-line:vertical,
        QComboBox QAbstractItemView QScrollBar::sub-line:vertical {
            height: 0px;
        }</string>
         </property>
         <property name="iconSize">
          <size>
           <width>10</width>
           <height>10</height>
          </size>
         </property>
         <item>
          <property name="text">
           <string>Seleccione rol</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Empleado</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Administrador</string>
          </property>
         </item>
        </widget>
       </widget>
      </item>
      <item alignment="Qt::AlignmentFlag::AlignHCenter">
       <widget class="QPushButton" name="btnRegistrar_user">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>35</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>35</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Segoe UI</family>
          <pointsize>13</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LayoutDirection::LeftToRight</enum>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>Registrar</string>
        </property>
        <property name="icon">
         <iconset resource="../icons/resources_ui.qrc">
          <normaloff>:/icons/confirm_white.png</normaloff>:/icons/confirm_white.png</iconset>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignmentFlag::AlignHCenter">
       <widget class="QPushButton" name="btnCancelar_reg_user">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>35</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>35</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Segoe UI</family>
          <pointsize>13</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LayoutDirection::LeftToRight</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: #e74c3c;</string>
        </property>
        <property name="text">
         <string>Cancelar</string>
        </property>
        <property name="icon">
         <iconset resource="../icons/resources_ui.qrc">
          <normaloff>:/icons/cancelar_w2.png</normaloff>:/icons/cancelar_w2.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../icons/resources_ui.qrc"/>
  <include location="../icons/resources_ui.qrc"/>
 </resources>
 <connections/>
</ui>
